HelgaShiryaeva/assignment1/ReversedListAccessUtil

```diff
@@ -0,0 +1,91 @@
+public class ReversedListAccessUtil {
+    public <T> T getReversed(int index, MyList<T> list) {
+      return list.get(list.size() - index);
+    }
+}
+class MyList<T> {
```

Comment:
It would be better to move this class to a separate file, since it is part of the interface of your package and should be public - other packages cannot use `ReversedListAccessUtil.getReversed` if `MyList` is hidden as package private.